ush/wave_domain_grid.sh

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env bash                                                                                                                                                                          
+
+################################################################################
+## UNIX Script Documentation Block
+## Script name:         wave_domain_grid.sh
+## Script description:  provides the wave grid specific values that
+##                      are needed for the wave related jobs
+#######################
+# Main body starts here
+#######################
+
+process_grdID() {
```

Comment:
@AntonMFernando-NOAA  this is a really big improvement -  Thank you!!! 

Thinking about the future based on a conversation with @WalterKolczynski-NOAA  @sbanihash and others, this is one place where gfs vs gefs might have different values.  I'm assuming we'd possibly be able to allow for that here eventually if needed?   Also, for exgfs_wave_post_gridded_sbs.sh we'll possibly need more than grdNAME passed back to the other routines --- I'm just noting that only 1 variable is exported here, so just double checking... We might have to call them "WAV_"  to not interfere with other things potentially

Reply:
Thanks @JessicaMeixner-NOAA for the comment. If gefs and gfs have different values they can be added with different functions or modify the existing function in `ush\wave_domain_grid.sh`.
One question.
//We might have to call them "WAV_"//
You mean the `gedNAME`  or all the variables in the `process_grdID()`?

Reply:
oh - actually now that i look at this more closely, it looks like only 1 routine is calling this. I thoght that archive and other scripts might also call this and then get variables confused.  Nevermind...